server/graphql/invoice_line/src/mutations/outbound_shipment_line/unallocated_line/update.rs

```diff
@@ -92,8 +93,14 @@ fn map_error(error: ServiceError) -> Result<UpdateErrorInterface> {
         ServiceError::LineDoesNotExist => {
             return Ok(UpdateErrorInterface::RecordNotFound(RecordNotFound {}))
         }
+        ServiceError::InvoiceDoesNotExist => {
```

Comment:
Yep I think as per @clemens-msupply those are internal errors (it would never actually happen), one way to avoid this is to use InvoiceLineRepository in check_line_exists_option, this would return InvoiceLine with both invoice line row and invoice row